src/App.tsx

```diff
@@ -1,14 +1,39 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { SelectCategory } from './types/SelectCategory';
+import { Todo } from './types/Todo';
+import { filterTodos } from './components/utils/filterTodos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [query, setQuery] = useState('');
+  const [selectTodoId, setSelectTodoId] = useState<number | null>(null);
+  const [selectCaregory, setSelectCategory] = useState<SelectCategory>(
```

Comment:
There is a typo in the state variable name `selectCaregory`. It should be `selectCategory` to match the naming convention and the expected usage in the `filterTodos` function.